php/installer/Installer.class.inc

```diff
@@ -248,9 +248,6 @@ class Installer
         if ($this->_sourceSchemaFile($DB, "04-Help.sql") == false) {
             return false;
```

Comment:
This doesn't seem to be a part of the PR, make sure to branch out of a fresh 17.1 branch